pkgs/development/tools/electron/common.nix

```
@@ -247,6 +254,12 @@ in
     mkdir -p $libExecPath
     unzip -d $libExecPath out/Release/dist.zip
 
+    # Create reproducible tarball, per instructions at https://reproducible-builds.org/docs/archives/
+    tar --sort=name \
+      --mtime="@$SOURCE_DATE_EPOCH" \
+      --owner=0 --group=0 --numeric-owner \
+      -czf $headers -C out/Release/gen node_headers
```

```suggestion
      -cf "$headers" --gzip --directory="out/Release/gen" node_headers
```

or even 

```suggestion
      --create --file="$headers" --gzip --directory="out/Release/gen" node_headers
```

Long forms are usually easier to read and understand and as such preferred.

Not blocking, just something to keep in mind in the future :)